docs/src/modules/spring/pages/actions.adoc

```diff
@@ -1 +1,73 @@
 = Implementing Actions
+
+include::ROOT:partial$include.adoc[]
+include::partial$actions.adoc[]
+
+In this first example, you will learn how to implement an Action as a pure stateless function. You will create a `FibonacciAction` that takes a number and returns the
+next number in the Fibonacci series.
+
+== Implementing the Action
+
+To implement this action you need the following:
+
+- Extend our class from `kalix.javasdk.action.Action`. This is generic. No matter what action you want to create you always need to extend from link:{attachmentsdir}/testkit/kalix/javasdk/action/Action.html[`Action` {tab-icon}, window="new"]. The rest is specific to this case
```

Comment:
An action doesn't need to have REST annotations. You can have an Action that subscribes to the event journal and publish messages to a Topic. 

In such a case, there is now REST endpoint and the Action is only accessible internally from the Proxy.

Reply:
But I agree that we can remove the sentence "The rest is specific to this case". 

It's not relevant to say it IMO. When we show other usages of Action, we can mention that a REST annotation is useless, for example.

Reply:
> An action doesn't need to have REST annotations. You can have an Action that subscribes to the event journal and publish messages to a Topic.
> 
> In such a case, there is now REST endpoint and the Action is only accessible internally from the Proxy.

Ah nice, indeed! 👍

Reply:
I think is better to add it here. In this example so the reader knows it straight away. Instead of letting them think that this may be required. Just as @efgpinto thought it was. But as you both agreed I'll remove it.

Reply:
> I think is better to add it here. In this example so the reader knows it straight away. Instead of letting them think that this may be required. Just as @efgpinto thought it was. But as you both agreed I'll remove it.

Yeah, understand why you think it's a good note. I wouldn't mind having it as a `Note:` instead maybe after ennumerating the steps saying that in this case we want the action to be exposed but if that's not the case, the first step is the only mandatory or something along those lines.

Reply:
Alrighty. Let's do that in another section later.